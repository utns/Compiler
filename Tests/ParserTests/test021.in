var
    a: integer;
    
procedure foo();
    procedure foo();
    begin
        a := 1;
    end;
begin
    foo();
end;

begin
    foo();
end.
