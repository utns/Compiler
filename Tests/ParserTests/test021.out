begin
____1
:=
____a
end

begin
(
__foo
end

begin
(
__foo
end
